specs/composer/show_all_packages.yml

```diff
@@ -0,0 +1,11 @@
+---
+name: Show a list of available Composer packages
```

Comment:
This and other Composer commands were updated to reflect feedback on the initial code review that was closed by Rob: https://github.com/warpdotdev/workflows/pull/94/files